desktop-widgets/divelistview.cpp

```diff
@@ -831,15 +831,11 @@ void DiveListView::shiftTimes()
 void DiveListView::loadImages()
 {
 	QStringList m_filters = mediaExtensionFilters();
-	QStringList i_filters = imageExtensionFilters();
-	QStringList v_filters = videoExtensionFilters();
 	QStringList fileNames = QFileDialog::getOpenFileNames(this,
 							      tr("Open media files"),
 							      lastUsedImageDir(),
-							      QString("%1 (%2);;%3 (%4);;%5 (%6);;%7 (*.*)")
+							      QString("%1 (%2);;%3 (*.*)")
```

Comment:
Since you're touching this line - this should probably be `QStringLiteral(...)`. Not that it matters, but out of principle...